wp-content/themes/pub/wporg-learn-2020/css/settings/_colors.scss

```diff
@@ -41,3 +41,7 @@ $color__text-on-dark: #fff;
 $color__text-heading: $color-gray-400;
 $color__text-heading-darker: $color-gray-800;
 $color__link-button: $color-gray-500;
+
+// Post type colors
+$color__workshops: #389547;
+$color__lesson-plans: #00669b;
```

Comment:
Looks like these are meant to map [to these brand colors](https://codepen.io/hugobaeta/pen/RNOzoV) — I think it would be better to add these vars above the theme colors section, and use them in the CSS instead.

```scss
$color-accent-blue-shade4: #006899;
$color-accent-green-shade1: #399648;
```

Reply:
Done.